src/core/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(EspressoCore_SRC
+  abs.cpp
```

Comment:
Is this related?

Reply:
yes its the newly introduced “anti blockading system” for particles

Reply:
just for making the build fail